validator/specs/lib/validateB3dmSpec.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
+var Cesium = require('cesium');
 var validateB3dm = require('../../lib/validateB3dm');
 
+var loadJson = Cesium.loadJson;
+
 describe('validateB3dm', function() {
+    var batchSchema;
+    beforeAll(function(done) {
+        console.log("beforeAll");
 
+        loadJson('https://raw.githubusercontent.com/AnalyticalGraphicsInc/3d-tiles/master/schema/batchTable.schema.json')
+            .then(function(schema) {
```

Comment:
I tried to change the validate files to take the batch table schema in as a parameter to avoid having to change everything  into an asynchronous process, as suggested previously. However, it seems like loadJson() is returning the "XMLHttpRequest i undefined" error. I have tried both the url above and this url: "https://github.com/AnalyticalGraphicsInc/3d-tiles/blob/master/schema/batchTable.schema.json" to load the Json. Am I not using this function properly?